### PR TITLE
add documentation to README for other view generators

### PR DESCRIPTION
**Pull request by esmale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,24 @@ For example, running the following will copy over all of the views into your app
...
+rails g madmin:views:navigation
+ # -> app/views/madmin/_navigation.html.erb
+ 
+rails g madmin:views:layout  # Note the layout generator includes the layout, javascript, and navigation files.
+ # -> app/views/madmin/application.html.erb
+ # -> app/views/madmin/_javascript.html.erb
+ # -> app/views/madmin/_navigation.html.erb
+```
+
+Running the views g
```
Comment (afomera) [inline README.md]: It appears this sentence is either unfinished, or no longer needed?

Comment (esmale) [reply README.md]: That's embarrassing.  Removed it.

Comment (afomera) [reply README.md]: Just chalk it up to being Friday evening after a long week 😀

**Comment by afomera** [comment]
Thanks @esmale! This looks pretty good to me, but I've learned to not merge code at 10PM, so I'll leave it for either a braver than me @excid3, or triple check it in the morning 😀

**Comment by excid3** [comment]
Lolol @afomera truth

**Pull request merged**